dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/security/jersey/UserAuthInjectable.java

```diff
@@ -52,7 +52,7 @@
   /**
    * Constants.
    */
-  public static final String AUTH_BEARER_TYPE = "Bearer";
+  public static final String APP_TOKEN_PREFIX = "Basic ";
```

Comment:
why did we need to change this from Bearer to Basic?

Reply:
Changing it back to Bearer.